Adafruit_AMG88xx.cpp

```diff
@@ -219,27 +219,25 @@ uint8_t Adafruit_AMG88xx::read8(byte reg) {
   return ret;
 }
 
-void Adafruit_AMG88xx::_i2c_init() { Wire.begin(); }
-
 void Adafruit_AMG88xx::read(uint8_t reg, uint8_t *buf, uint8_t num) {
-  uint8_t value;
   uint8_t pos = 0;
+  uint8_t buffer[1];
+  uint8_t read_buffer[AMG_I2C_CHUNKSIZE];
```

Comment:
btw you can ask busio what is the i2c max buffer size

Reply:
ah. cool. wanted to, but felt like i was going against some hard won dark magic, didn't want to mess with the chuckification, etc.